hugegraph-common/src/main/java/org/apache/hugegraph/rest/RestClientConfig.java

```diff
@@ -24,15 +24,16 @@
 @Builder
 @Getter
 @Setter
-public class OkHttpConfig {
+public class RestClientConfig {
 
     private String user;
     private String password;
     private String token;
     private Integer timeout;
-    private Integer maxTotal;
-    private Integer maxPerRoute;
-    private Integer idleTime;
+    private Integer maxConns;
+    private Integer maxConnsPerRoute;
+    private Integer idleTime = 5;
```

Comment:
its unit is in seconds? we expect a default value > 10s like 30s, but  I think we don't need to set a default value here.